proto/sources.proto

```diff
@@ -97,6 +97,8 @@ message Confluence {
   bool insecure_skip_verify_tls = 6;
   repeated string spaces = 7;
   repeated string ignore_spaces = 8;
+  bool include_attachments = 9;
```

Comment:
By default this will not include attachments. Do we want the default to include them or exclude them?

Reply:
This was intentional since it was a recent addition with major performance implications

Reply:
We'll be updating the docs soon to document all of the available options which will help discovery